packages/patternfly-4/src/components/_core/Documentation/index.js

```diff
@@ -57,7 +57,7 @@ export default class Documentation extends React.Component {
                 <SEO title="HTML" />
                 <PageSection variant={PageSectionVariants.light} className="pf-site-background-medium">
                   <AutoLinkHeader size="md" is="h1" className="pf4-site-framework-title">HTML</AutoLinkHeader>
-                  <AutoLinkHeader size="4xl" is="h2" className="pf-u-mt-sm pf-u-mb-md">{heading}</AutoLinkHeader>
+                  <AutoLinkHeader size="4xl" is="h2" id={`${heading.replace(/ /g, '-')}-title`} className="pf-u-mt-sm pf-u-mb-md">{heading}</AutoLinkHeader>
```

Comment:
The AutoLinkHeader generates the ids from the text, so this would break that

Reply:
That's a really good point, thanks! I updated it, let me know what you think. :)